packages/components/styles/_forms-variables.scss

```diff
@@ -0,0 +1,11 @@
+// General purpose tokens
```

Comment:
It'll be nice to have these for the other form components we port over / make in the future